test/tests/cli.js

```diff
@@ -728,26 +643,26 @@ function initDat(opts, cb) {
     if (error || stdo.indexOf('Initialized dat store') === -1) {
       throw error || stdo.toString()
     }
-    
+
     // dont serve when in rpc mode
     if (opts.rpc) return done()
-    
-    var server = spawn(datCliPath, ['listen'], opts)
-    
+
+    var server = spawn('dat-server', ['listen'], opts)
```

Comment:
oh, that makes sense. maybe it could be optional:

`datServer('./path/to/dat')` or `datServer(dat)`